### DISCUSSION TITLE
It's the small victories ...

### DISCUSSION THREAD

**Discussion by SirNibble** [prirun/p50em#26] [General]
Fed-up of trying to remember where I'd seen a file when hunting through the various Guest, Jimmy & Dennis UFDs, I finally remembered some _abbrev-fu_ ...

```
[017] sman>ab -list find
    FIND     ld *>@@>%1%@@ -wlkfm 1 -dm

[018] sman>find life.pas

<USERS>SMAN>PASCAL_PROJECTS>LIFE (ALL access)
32 records in this directory, 32 total records out of quota of 0.

1 File.

LIFE.PASCAL


<USERS>SMAN>BACKUP>LIFE (ALL access)
32 records in this directory, 32 total records out of quota of 0.

1 File.

LIFE.PASCAL

[019] sman>                                                                     
```

**Comment by drboone** [comment]
WHEREIS might be of use to you.

https://sysovl.info/pages/blobs/prime/boris/whereis.tap

**Comment by SirNibble** [comment]
Add getting ECL and EMACS to understand the concept of VT100 cursor keys to the list.

I don't think I ever worried about it back-in-the-day, but the overwhelming muscle-memory for 'regular' movement these days is too much. ;)

In fact ... I don't think I ever did much with EMAC's macros, et al, so a bit of an adventure trolling through the Writing & Ref guides. 

```
(set_permanent_key "^[[A" "prev_line_command")
(set_permanent_key "^[[B" "next_line")
(set_permanent_key "^[[C" "forward_char")
(set_permanent_key "^[[D" "back_char")

```
